balancer-labs/frontend-v2
# Correct impact message on invest/withdraw

# Description
Was testing deposits to the Lido pool and noticed:

1) The price impact checkbox says "withdrawing," even on the invest form
2) There are no translations for all the new Lido-related messaging

## Type of change


## How should this be tested?

Try to invest single token with >1% price impact: should say "depositing"
Try to withdraw single token with >1% price impact: should say "withdrawing," as before
Lido messaging is translated

## Checklist:
